docs/source/submission_strategies.rst

```diff
@@ -124,3 +124,109 @@ Command:
 .. code-block:: bash
 
     $ jade submit-jobs --num-processes=36 --time-based-batching config.json
+
+.. _submission_group_strategy:
+
+Jobs that require different submission parameters
+=================================================
+Some jobs will take less than 4 hours, and so can run on the short queue. Other
+jobs take longer and so need to run on the standard queue.
+
+Strategy:
+
+- Define two submission groups.
+- Set the submission group for each job appropriately.
+
+The ``SubmissionGroup`` object contains a name as well as a
+``SubmitterParameters`` object. The latter contains batch parameters like
+``per-node-batch-size`` as well as HPC parameters. You can customize most of
+these parameters for each submission group.
+
+Here's how to create the JSON object that you will need to add to the
+``config.json`` file.
+
+1. Create default submission parameters with ``jade config submitter-params -c submitter_params.json``.
```

Comment:
We could make a new command that creates a JSON blob with only the relevant fields.